src/vue/pages/Assets.vue

```diff
@@ -88,6 +88,9 @@ export default {
       vuexTypes.myBusinesses,
       vuexTypes.accountId,
     ]),
+    isAccountHaveBusiness () {
```

Comment:
Variables which start from 'is' mast return Boolean type of result.

Reply:
If account corporate, it already has business. You need to check does it have owned assets. Use ownedAssets getter.

Reply:
remove this

Reply:
removed